### PR TITLE
feat(webpack-model): add context to query method

### LINKED ISSUE #73: Pass context to `query`-method
```js
const j = prepareWithJora(stats);
j.query('foo', data, context);
```

### PR DESCRIPTION
**Pull request by immitsu**
Resolves #73

### CODE REVIEW

File: packages/webpack-model/src/index.ts [hunk 1]
```diff
@@ -32,11 +32,11 @@ export function prepareWithJora(
     ...options.helpers,
     ...joraHelpers(compilations),
   });
-  const context = {};
 
   return {
     files,
     compilations: compilations.map((c) => c.data),
-    query: (query: string, data: unknown): unknown => j(query)(data || files, context),
+    query: (query: string, data: unknown, context: unknown = {}): unknown =>
```
Comment (smelukov) [inline packages/webpack-model/src/index.ts:39]: I thank that creating a new object every time that we calling `query` without a context - not a good idea )
```suggestion
    query: (query: string, data: unknown, context: unknown = rootContext): unknown =>
```

File: packages/webpack-model/src/index.ts [hunk 2]
```diff
@@ -36,7 +36,7 @@ export function prepareWithJora(
   return {
     files,
     compilations: compilations.map((c) => c.data),
-    query: (query: string, data: unknown, context: unknown = {}): unknown =>
+    query: (query: string, data: unknown, context: unknown = rootContext): unknown =>
```
Comment (smelukov) [inline packages/webpack-model/src/index.ts:39]: You're forgot `const rootContext = {};` before `return` ;)

Comment (immitsu) [reply packages/webpack-model/src/index.ts:41]: Sorry! Thank you for your patience. And for the great tool too :)

File: packages/webpack-model/src/index.ts [hunk 3]
```diff
@@ -32,6 +32,8 @@ export function prepareWithJora(
     ...options.helpers,
     ...joraHelpers(compilations),
   });
+
```
Comment (smelukov) [inline packages/webpack-model/src/index.ts:35]: Please, remove two spaces from the start of the line to pass the linter

**Comment by smelukov** [comment]
Thanks!

**Pull request merged**